docs/minikube-tutorial.md

```diff
@@ -0,0 +1,196 @@
+  * [Delete TiDB cluster](#delete-tidb-cluster)
+- [FAQs](#faqs)
+  * [TiDB cluster in minikube is not responding or responds slow](#tidb-cluster-in-minikube-is-not-responding-or-responds-slow)
+
+## Start a Kubernetes cluster with minikube
+
+### What is minikube?
+
+[Minikube](https://kubernetes.io/docs/setup/minikube/) can start a local
+Kubernetes cluster inside a VM on your laptop. It works on macOS, Linux, and
+Windows.
+
+### Install minikube and start a Kubernetes cluster
+
+See [Installing Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) to install
+minikube on your machine.
+
+After you installed minikube, you can run the following command to start a
+Kubernetes cluster.
+
+```
+minikube start
+```
+
+For Chinese mainland users, you may use local gcr.io mirrors such as
+`registry.cn-hangzhou.aliyuncs.com/google_containers`.
+
+```
+minikube start --image-repository registry.cn-hangzhou.aliyuncs.com/google_containers
+```
+
+or configure HTTP/HTTPS proxy environments in your docker, e.g.
+
+```
+# change 127.0.0.1:1086 to your http/https proxy server IP:PORT
+minikube start --docker-env https_proxy=http://127.0.0.1:1086 \
+  --docker-env http_proxy=http://127.0.0.1:1086
+```
+
+See [minikube setup](https://kubernetes.io/docs/setup/minikube/) for more options to
+configure your virtual machine and Kubernetes cluster.
+
+### Install kubectl to access the cluster
+
+The Kubernetes command-line tool,
+[kubectl](https://kubernetes.io/docs/user-guide/kubectl/), allows you to run
+commands against Kubernetes clusters.
+
+Install kubectl according to the instructions in [Install and Set Up kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
+
+After kubectl is installed, test your minikube Kubernetes cluster:
+
+```
+kubectl cluster-info
+```
+
+## Install TiDB operator and run a TiDB cluster with it
+
+### Install helm
+
+Helm is the package manager for Kubernetes and is what allows us to install all of the distributed components of TiDB in a single step. Helm requires both a server-side and a client-side component to be installed.
+
+```
+curl https://raw.githubusercontent.com/helm/helm/master/scripts/get | bash
+```
+
+Install helm tiller:
+
+```
+helm init
+```
+
+If you have limited access to gcr.io, you can try a mirror, e.g.
+
+```
+helm init --upgrade --tiller-image registry.cn-hangzhou.aliyuncs.com/google_containers/tiller:$(helm version --client --short | grep -P -o 'v\d+\.\d+\.\d')
+```
+
+Once it is installed, running `helm version` should show you both the client
+and server version, e.g.
+
+```
+$ helm version
+Client: &version.Version{SemVer:"v2.13.1",
+GitCommit:"618447cbf203d147601b4b9bd7f8c37a5d39fbb4", GitTreeState:"clean"}
+Server: &version.Version{SemVer:"v2.13.1",
+GitCommit:"618447cbf203d147601b4b9bd7f8c37a5d39fbb4", GitTreeState:"clean"}
+```
+
+If it shows only the client version, `helm` cannot yet connect to the server. Use
+`kubectl` to see if any tiller pods are running.
```

Comment:
Add the related command here.

Reply:
done, `kubectl -n kube-system get pods -l app=helm` is added here.